plugin/pkg/scheduler/algorithm/priorities/taint_toleration_test.go

```diff
@@ -204,6 +204,18 @@ func TestTaintAndToleration(t *testing.T) {
 				{Host: "nodeC", Score: 0},
 			},
 		},
+		{
+			test: "Default behaviour No taints and tolerations, lands on node with no taints",
+			//pod without tolerations
+			pod: podWithTolerations([]v1.Toleration{}),
+			nodes: []*v1.Node{
+				//Node without taints
```

Comment:
I am slightly confused, are you telling that the node with taints should get a score of 0? It should not get 0 unless the effect is TaintEffectPreferNoSchedule? - Correct me if I am wrong and that case has been handled already.